pkg/sypgp/sypgp.go

```diff
@@ -564,6 +564,9 @@ func EncryptKey(k *openpgp.Entity, pass string) error {
 
 // SelectPubKey prints a public key list to user and returns the choice
 func SelectPubKey(el openpgp.EntityList) (*openpgp.Entity, error) {
+	if el == nil {
+		return nil, fmt.Errorf("no key in public keyring")
+	}
 	PrintPubKeyring()
```

Comment:
Just looked into this func, it reads file to fetch `openpgp.EntityList` to print, but we already have one in `SelectXXXKey` arguments... I think that should be refactored. Should we create a separate issue for that, what do you think?

Reply:
I know @mem and @gvallee was doing some refactoring on this package, I want to change as less as possible. And I would prefer to change that in a separate PR. (Small PRs are more likely to get merged quicker :smile:)